errors/B0004.md

```diff
@@ -0,0 +1,112 @@
+# B0004
+
+A runtime warning.
+
+An [`Entity`] with a hierarchy-inherited component has a [`Parent`]
+without the hierarchy-inherited component in question.
+
+The hierarchy-inherited components are:
+
+- [`ComputedVisibility`]
+- [`GlobalTransform`]
```

Comment:
The plugin can be used by users/crate authors so this warning may be emitted for other components as well.